### DISCUSSION TITLE
Outdated Conversations should have an option to automatically resolve

### DISCUSSION THREAD

**Discussion by Y0RI0** [community/community#66449] [Discussions]
### Select Topic Area

Product Feedback

### Body

There are instances where CICD with control over PR merging won't merge unless conversations are resolved. But quite frequently there is an outdated conversation that is no longer relevant since new commits have been pushed that blocks a PR merge and requires a manual resolution and reset in order for the CICD application to automerge. One such example is [atlantis](https://github.com/runatlantis/atlantis).

Since stale approvals can be automatically dismissed. It would be nice to have a sub-option under:
`Require conversation resolution before merging`  along the lines of 
`Automatically resolve outdated conversations`, which is disabled by default.

**Comment by Biancakalutor** [comment]
https://github.com/community/community/blob/e9598b0c7e750d8d225ac3eed9306369304d7556/CODE_OF_CONDUCT.md

**Reply by Biancakalutor** [reply]
[](url)

**Reply by Biancakalutor** [reply]
https://github.com/github/feedback/discussions/4374

**Comment by Biancakalutor** [comment]
[](url)
